sklearn/naive_bayes.py

```diff
@@ -893,19 +893,19 @@ class BernoulliNB(_BaseDiscreteNB):
 
     Parameters
     ----------
-    alpha : float, optional (default=1.0)
+    alpha : float, default=1.0
         Additive (Laplace/Lidstone) smoothing parameter
         (0 for no smoothing).
 
-    binarize : float or None, optional (default=0.0)
+    binarize : float or None, default=0.0
```

Comment:
`float, default=0.0`

Reply:
Looks like parameter can be either float or None here. I didn't find documentation saying that None should be removed from docstring in these cases.

Reply:
you have a point, i think best way is `float, default=None` but feel free to let the developers decide this one.

Reply:
Ok, I left it as is, feel like it is better to have it stated explicitly.
I have fixed all other instances of `array, shape`, `array-like, shape` and square brackets